Add disclaimer (new > old release) (#70)
volkamerlab/teachopencadd: README.md

[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.2600909.svg)](https://doi.org/10.5281/zenodo.2600909)
[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/volkamerlab/TeachOpenCADD/master)

> > ⚠ You are looking at a new release of the TeachOpenCADD repository with [major changes](https://github.com/volkamerlab/teachopencadd/pull/29).  

Reviewer: Maybe a warmer emoji? We want to them love this, not to be scared heh

```suggestion
> 🌟 You are looking at a new release of the TeachOpenCADD repository with [major changes](https://github.com/volkamerlab/teachopencadd/pull/29).  
```

Author: Great! Where can I find a list of these icons?

> > ⚠ You are looking at a new release of the TeachOpenCADD repository with [major changes](https://github.com/volkamerlab/teachopencadd/pull/29).  

Reviewer 2: I was wondering here to were we want to point? because after merging the changes are in? maybe rather link to the new version? tbd

Author: Good idea. We are creating a new release tag after we did the `packaging` > `master` merge, correct @jaimergp? 
If that is the case, we could keep the link as-is in this PR and update the link when we are on master.

Author: Same goes for the link to our soon-to-be conda package (to be added to one of the disclaimer's bullet points):

Author: (I added both to this PR's TODOs - to be moved around PRs as needed.)

>  - We applied a new folder structure to the repository allowing us to offer for the first time a `teachopencadd` conda package!

> >  - We refactored of all published notebooks (T001-T010) to apply best practices to our code!

Reviewer 2: ```suggestion
>  - We refactored all published notebooks (T001-T010) to apply best practices to our code!
```

>  - We proudly present our brand new [TeachOpenCADD website](https://projects.volkamerlab.org/teachopencadd/)!
>  
> We are adding new notebooks to the repository soon - 
> subscribe to the repository releases (top-right menu: `Watch` > `Releases`) to stay tuned!
>

> > If you would like to continue to work with the old repository structure, please refer to release [`1.3.0`](https://github.com/volkamerlab/teachopencadd/releases/tag/v1.3.0).

Reviewer: I would also point to the tree view of that commit. 
```suggestion
> If you would like to continue to work with the old repository structure, please refer to release [`1.3.0`](https://github.com/volkamerlab/teachopencadd/tree/v1.3.0). Downloadable [here](https://github.com/volkamerlab/teachopencadd/release/tag/v1.3.0).
```

> > If you would like to continue to work with the old repository structure, please refer to release [`1.3.0`](https://github.com/volkamerlab/teachopencadd/releases/tag/v1.3.0).

Reviewer 2: ```suggestion
> If you would like to continue working with the old repository structure, please refer to release [`1.3.0`](https://github.com/volkamerlab/teachopencadd/releases/tag/v1.3.0).
```

> > If you use TeachOpenCADD in a publication,

Reviewer: This second notice right after the previous one is hard to notice now. I don't know what to do about it, but for now maybe we can move it _after_ the description?

Reviewer 2: I had the same feeling and would suggest to move it below for as long as we are having the current disclaimer.

Author: Good point. I moved it between the description and figure.
